src/components/Attachments/AttachmentCarousel/index.js

```diff
@@ -207,7 +207,7 @@ function AttachmentCarousel({report, reportActions, source, onNavigate, setDownl
                             getItemLayout={getItemLayout}
                             keyExtractor={(item) => item.source}
                             viewabilityConfig={viewabilityConfig}
-                            onViewableItemsChanged={updatePage.current}
+                            onViewableItemsChanged={updatePage}
```

Comment:
There's bad regression on this line:


https://github.com/Expensify/App/assets/108292595/34c36dfe-e49e-4d38-8dcd-3980def33f40